soql-types/src/main/scala/com/socrata/soql/types/SoQLType.scala

```diff
@@ -77,6 +77,25 @@ case object SoQLID extends SoQLType("row_identifier") {
       obfuscator.obfuscate(soqlId)
   }
 
+  /**
+   * This exists for OBE compatibility reason.
```

Comment:
This is for presenting obe nbe id as obe sid.  We are not creating a new column, are we?